README.md

```diff
@@ -115,23 +99,23 @@ Which one of the following concepts does this best illustrate?
 What is an IIFE, and why might you use it?
 
 **Your answer:**
-
+standas for immediately invoked function expression, its good practice to use it because in very large scale projects, we do not want to pollute the name space in all of our controllers, meaning that there will be no naming conflicts in all the controllers and each controller has its own set of variables.
 > ...
 
 ## Question 7
 
 What is the `ui-sref` directive, and how is it used?
 
```

Comment:
not exactly,  I think you are thinking of `ui-view`, `ui-sref` is a way of making links to other states